tdp/cli/commands/init.py

```diff
@@ -31,19 +36,49 @@
     multiple=True,
     help="Path to TDP variables overrides. Can be used multiple times. Last one takes precedence.",
 )
+@click.argument(
+    "alembic_ini_path",
+    envvar="ALEMBIC_CONFIG",
```

Comment:
Why not add the `TDP_` prefix for this one?

Reply:
Resolved it another way. Gave the absolute path of the tdp-lib folder